.github/workflows/build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build and Test
+
+on:
+    push:
+      branches:
+        - master
+    pull_request: {}
+
+jobs:
+    build:
+        name: Build and Test
+        runs-on: macos-latest
+        steps:
+
+        - name: Checkout repository
+          uses: actions/checkout@v2
+
+        - name: Cache bazel build results
+          uses: actions/cache@v2
+          env:
+            cache-name: bazel-cache
+          with:
+            path: ~/.cache/bazel
+            key: ${{ runner.os }}-${{ env.cache-name }}-${{ github.ref }}
+            restore-keys: |
+              ${{ runner.os }}-${{ env.cache-name }}-master
```

Comment:
It doesn't [appear to work](https://github.com/Snapchat/djinni/runs/4220359952?check_suite_focus=true) since the build times for my PR are >5min. Have you looked at the example actions config using `bazelisk` https://github.com/marketplace/actions/setup-bazelisk?